mods/ra/rules/infantry.yaml

```diff
@@ -507,6 +513,8 @@ SHOK:
 	Armament@GARRISONED:
 		Name: garrisoned
 		Weapon: PortaTesla
+	TakeCover:
+		ProneOffset: 500,0,-341
```

Comment:
This is too far forward too.  Make sure that you use the combat debug overlay and compare the muzzle position before and after going prone.